shims/spark310/src/main/scala/com/nvidia/spark/rapids/shims/spark310/Spark310Shims.scala

```diff
@@ -111,17 +111,17 @@ class Spark310Shims extends Spark301Shims {
         }),
     GpuOverrides.expr[AnsiCast](
       "Convert a column of one type of data into another type",
-      new CastChecks() {
+      new CastChecks {
         import TypeSig._
         // nullChecks are the same
 
         override val booleanChecks: TypeSig = integral + fp + BOOLEAN + STRING
         override val sparkBooleanSig: TypeSig = numeric + BOOLEAN + STRING
 
-        override val integralChecks: TypeSig = integral + fp + BOOLEAN + STRING
+        override val integralChecks: TypeSig = numeric + BOOLEAN + TIMESTAMP + STRING + BINARY
```

Comment:
Sorry this is a really minor nit. Spark does not support TIMESTAMP or BINARY (as you can see on the line below), so us claiming to support it is technically wrong.  This should not impact the generated docs or the correctness of the code in any way.

It is also an issue for fpChecks with TIMESTAMP.

The only reason to fix it is to avoid confusion.

Reply:
It's been fixed. My bad :)